de/surveys/data_privacy_day.lang

```diff
@@ -294,13 +294,13 @@ Meine Privatsphäre zu berücksichtigen, wenn ich einkaufe.
 
 
 # q-25
-;Email Address - We’ll use this to contact you if/when Mozilla has privacy or security information we think you’d find interesting.
-E-Mail-Adresse – Darüber kontaktieren wir Sie, wenn Mozilla Informationen zu Datenschutz oder Sicherheit hat, die Sie interessieren könnten.
+;Email Address (optional) - We’ll use this to contact you if/when Mozilla has privacy or security information we think you’d find interesting.
+E-Mail-Adresse (freiwillig) – Darüber kontaktieren wir Sie, wenn Mozilla Informationen zu Datenschutz oder Sicherheit hat, die Sie interessieren könnten.
```

Comment:
@koehlermichael just a heads up, asked Aryx about this translation, to avoid retranslating the string